plantuml/LinkLoss/data_written_normal.puml

```diff
@@ -0,0 +1,18 @@
+@startuml
+participant LinkLossService as LLS
+participant Gap as GAP
+participant GattWriteAuthCallbackParams as GWACP
+
+== Link Loss Service Constructed ==
+
+LLS <- GAP : onDataWritten(write_request)
+activate LLS
+
+LLS -> GWACP : level = write_request->data;
```

Comment:
We probably don't need that level of details with the GattWriteAuthCallback param. 
If there's one internal process we want to model it is the data validation. No need to be explicit about that, another diagram can be made when the data in input are not valid.